conf/msg_conf/map_msg_chn.conf

```diff
@@ -871,9 +871,9 @@
 // @mapinfo
 1038: 請至少輸入一個有效的列表數字 (使用方法: @mapinfo <0-3> [地圖名稱])
 1039: ------ 地圖資訊 ------
-1040: 地圖名: %s | 地圖玩家數量: %d | 地圖NPC數量: %d | 地圖聊天室數量: %d
+1040: 地圖名: %s (Zone: %s) | 地圖玩家數量: %d | 地圖NPC數量: %d | 地圖聊天室數量: %d | Vendings: %d
 1041: ------ 地圖旗標 ------
-1042: 城市地圖
+1042: Battle Rate Adjustments:
```

Comment:
Why is battle instead of damage?

Reply:
I guess I was thinking overall "battle" adjustments but can adjust to damage to be a bit more clear!

Reply:
I mean because you used 'damage' on the english file